### ISSUE TITLE
Colab Training Problem

### ISSUE THREAD

**Issue by zhanxinrui** [phil-bergmann/tracking_wo_bnw#55]
Is your detection model trained with same codes link you provide in colab? I can't obtain the same AP & Prec in 27epoch with those codes.

**Comment by timmeinhardt** [comment]
Yes, I used the same Colab notebook but I trained longer and changed the learning rate manually in between. If you want to retrain the object detector on a new dataset, you have to find the respective optimal hyperparameters, e.g., number of epochs and learning rate scheduling.

**Comment by zhanxinrui** [comment]
I'll have a try. Thanks a lot!

**Comment by timmeinhardt** [comment]
Good luck! Can we close this issue?
